.github/workflows/ci-license.yaml

```diff
@@ -74,12 +74,12 @@ jobs:
           java-version: 11
 
 
-      # license check fails with 3.6.2 so we have to downgrade
+      # The plugin org.apache.nifi:nifi-nar-maven-plugin:1.5.0 requires Maven version 3.6.3
       - name: Set up Maven
         if: ${{ steps.check_changes.outputs.docs_only != 'true' }}
         uses: apache/pulsar-test-infra/setup-maven@master
         with:
-          maven-version: 3.6.1
```

Comment:
I would assume that we could remove this completely if we upgrade license-maven-plugin.version to `4.1` in pom.xml.

Reply:
@RobertIndie I pushed the changes directly to this PR to expedite fixing this together.